### PR TITLE
Add example data to CuratedLink form

### PR DESCRIPTION
**Pull request by garethrees**
Give examples of `CuratedLink` data so that admins have a better sense
of what they need to input.

screen shot 2018-06-01 at 16 26 29

### CODE REVIEW

**Comment by zarino** [comment]
#116 modifies the structure of these `curated_link` admin pages, putting the form inside a half-width container. So if you rebase this branch on top of `26-curated-link-styling`, the form hints will be limited to the same width as the inputs they relate to – rather than extending wider, as in the screenshot above. :+1:

**Comment by garethrees** [comment]
Rebased and updated screenshot

**Comment by zarino** [comment]
Thanks for the updated screenshot.

I’ve had a second think about this. The examples just seem far too wordy and it’s not really clear to me what they‘re trying to communicate.

I’d be tempted to remove the `form-hint` under URL, Title, and Keywords, and then change the Summary one to something that describes _what_ in particular we’re trying to guide them towards writing.

So maybe something like:

> **Summary**
> Describe the information available at this URL, in words a layperson would understand. Aim for around 20-30 words.

The example under Keywords is kind of useless once the Taggle component has loaded, because it‘s no longer an accurate example of what you should type into the box. But it’s useful to have there if the Taggle component _hasn’t_ loaded, because it shows the format you need to follow for the manual CSV input. Maybe we keep the `form-hint` there, but hide it once the JavaScript loads?

**Comment by zarino** [comment]
@gbp I’ve updated this PR. Main changes:

* Only show hints for the Summary and Keywords fields, because they’re the least obvious ones.
* Swap the Keywords hint to something Taggle-specific, once Taggle has loaded.

What do you think? Is there some pre-built way of getting a localised string into the javascript, rather than hard-coding it?

screen shot 2018-06-12 at 16 21 29

**Comment by zarino** [comment]
> This may not be the PR to address this in, but it's a bit annoying that if you enter one keyword and don't hit 'enter', and then submit the form, the keyword doesn't get stored.

Will fix!

**Comment by zarino** [comment]
@crowbot I’ve added the save on blur functionality in a separate PR: https://github.com/mysociety/foi-for-councils/pull/134
